src/components/ChangePlan/ChangePlan.js

```diff
@@ -1,58 +1,145 @@
 import { Helmet } from 'react-helmet';
 import { Card, CardPrice, CardAction, Ribbon, CardFeatures } from './Card';
 import queryString from 'query-string';
 import { extractParameter } from '../../utils';
-
-function getPlans() {
-  return [
-    {
-      price: 15,
-      quantity: 1500,
-      name: 'Standard',
-      id: 2,
-      EmailParameterEnabled: true,
-      CancelCampaignEnabled: true,
-      SiteTrackingLicensed: false,
-      SmartCampaignsEnabled: false,
-      ShippingLimitEnabled: false,
-      PlanType: 2,
-    },
-    {
-      price: 21,
-      quantity: 1500,
-      name: 'Plus',
-      id: 1,
-      EmailParameterEnabled: true,
-      CancelCampaignEnabled: true,
-      SiteTrackingLicensed: true,
-      SmartCampaignsEnabled: true,
-      ShippingLimitEnabled: true,
-      PlanType: 3,
-    },
-  ];
-}
+import { InjectAppServices } from '../../services/pure-di';
 
 function getPlanUrl(planId, advancedPay, promoCode, _) {
   return (
     _('common.control_panel_section_url') +
     `/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=${planId}&fromStep1=True&IdDiscountPlan=${advancedPay}&PromoCode=${promoCode}`
   );
 }
+const BulletOptions = ({ type }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const starBullet = () => {
+    return (
+      <span className="dp-icostar">
+        <img alt="star icon" src={_('common.ui_library_image', { imageUrl: 'ico-star.svg' })} />
+      </span>
+    );
+  };
+
+  const newLabel = () => {
+    return dopplerLabel(_('change_plan.new_label'));
+  };
+
+  return (
+    <FormattedMessage
+      id={'change_plan.features_HTML_' + type}
+      values={{
+        option: (chunks) => optionItem(chunks, basicBullet()),
+        star: (chunks) => optionItem(chunks, starBullet()),
+        newOption: (chunks) => optionItem(chunks, basicBullet(), newLabel()),
+        newStar: (chunks) => optionItem(chunks, starBullet(), newLabel()),
+        bigData: (chunks) => optionItem(chunks, bigDataBullet()),
+        newBigData: (chunks) =>
+          optionItem(chunks, bigDataBullet(_('change_plan.big_data_tooltip')), newLabel()),
+      }}
+    >
+      {(txt) => <ul className="dp-list-detail">{txt}</ul>}
+    </FormattedMessage>
+  );
+};
+
+const optionItem = (chunks, bullet, label) => {
+  return (
+    <li>
+      {bullet}
+      <span>
+        {chunks} {label}
+      </span>
+    </li>
+  );
+};
 
-const ChangePlan = ({ location }) => {
+const basicBullet = () => {
+  return <span className="dp-icodot">.</span>;
+};
+
+const dopplerLabel = (text) => {
+  return <span className="dp-new">{text}</span>;
+};
+
+const bigDataBullet = (tooltipText) => {
+  return (
+    <div className="dp-tooltip-container">
+      <span className="dp-icobd">BD</span>
+      <div className="dp-tooltip-block">
+        <span className="tooltiptext">{tooltipText}</span>
+      </div>
+    </div>
+  );
+};
```

Comment:
Esto es raro, por que son funciones y no componentes?

Reply:
I think of this functions as components, it´s true I'm using them as functions I will fix it.

Reply:
Creo que para usarlas como componentes los nombres tienen que comenzar con mayusculas, y los parámetros tienen que entra como props (o childs)